### ISSUE TITLE
Rotur Websocket is Broken

### ISSUE THREAD

**Issue by hammouda101010** [RoturTW/main#6]
in other words: Your Websocket is dead.

**Comment by ThePandaDever** [comment]
we know

**Comment by ThePandaDever** [comment]
servers down for some reason (mist is looking into it) and i dont think we can do anything until milo comes back so

**Comment by hammouda101010** [comment]
=       =
=====

**Comment by Mistium** [comment]
> just a small question: is it possible to make online games with badges, microtransactions (that don't use real money), friends and other features in Rotur ?

yes but rotur isnt intended for online games. badges and currency are controlled entirely by the system and not by the project, so you cannot give users any rotur credits. you would have to use account storage to make your own currency system.

tldr: yes, but rotur is meant for webos not for games, use cloudlink 5 if you want to make networked games

**Comment by Mistium** [comment]
@MiloDev123

**Comment by MiloDev123** [comment]
@Mistium no shit sherlock
